stamina-testkit/src/main/scala/stamina/testkit/StaminaTestKit.scala

```diff
@@ -0,0 +1,85 @@
+package stamina
+package testkit
+
+import scala.util._
+import scala.annotation.tailrec
+
+trait StaminaTestKit { self: org.scalatest.WordSpecLike ⇒
```

Comment:
In my experience duplicate class names in different packages add more confusion than clarity. I'd prefer explicit class names, even if we put them in separate package for other reasons.